spec/models/game_spec.rb

```diff
@@ -43,4 +43,50 @@
       expect(game.white_to_move).to eq(true)
     end
   end
+
+  describe "Populate Board" do
+    it "should have correct number of pieces" do
+      expect(game.pieces.count).to eq 32
+    end
+
+    it "should be empty on the squares" do
+      expect(game.pieces.where(rank: 3..6)).to eq([])
+    end
+
+    (1..8).each do |file|
+      it "should have a white pawn at rank 2, file #{file}" do
+        white_pawn = game.pieces.find_by(rank: 2 , file: file)
+        expect(white_pawn.present?).to eq(true)
+        
+        expected = ["Pawn", "white"]
+        expect([white_pawn.type, white_pawn.color]).to eq(expected)
+      end
+
+      it "should have a black pawn at rank 7, file #{file}" do
+        black_pawn = game.pieces.find_by(rank: 7 , file: file)
+        expect(black_pawn.present?).to eq(true)
+        
+        expected = ["Pawn", "black"]
+        expect([black_pawn.type, black_pawn.color]).to eq(expected)
```

Comment:
You're still using the problematic pattern that I mentioned here: https://github.com/fp-team-maniac/maniac-chess/pull/12#discussion_r60299653. Is this intentional or did you miss that part of the comment? Since this is a style point, I don't want to hold up the merge because of it, but I'd like to understand your reasoning.